sys/include/stdio_base.h

```diff
@@ -34,6 +36,18 @@ extern "C" {
  */
 void stdio_init(void);
 
+#if IS_USED(MODULE_STDIO_AVAILABLE) || DOXYGEN
+/**
+ * @brief   Get the number of bytes available for reading from stdio.
+ *
+ * @warning This is function is only available if the implementation supports
```

Comment:
```suggestion
 * @warning This function is only available if the implementation supports
```

Reply:
Should I squash it directly to avoid an additional cycle in Murdock?

Reply:
Yes, please squash!